test/mocha/10-test.js

```diff
@@ -3,9 +3,54 @@
  */
 'use strict';
 
-const {documentLoader, jsonLdDocumentLoader} =
+const {documentLoader, jsonLdDocumentLoader, httpClientHandler} =
   require('bedrock-jsonld-document-loader');
+const express = require('express');
+const fs = require('fs');
+const https = require('https');
 
+const TEST_SERVER_PORT = 5000;
```

Comment:
This could be auto-assigned.  Leave the port out of the `listen()` call, and use `server.address()` in the callback to get the port.  Not usually a big deal, but it does avoid ever having issues with ports being in use by other apps or tests.  BASE_URL would have to be updated too.

Reply:
Updated 152e81e

Reply:
As a data point, apparently I have this exact problem.  Just tried to run tests and docker-registry has 5000 open, so tests fail to run.

Reply:
TIL :) thank you!